app/workers/evss/disability_compensation_form/job_status.rb

```diff
@@ -117,7 +119,10 @@ def non_retryable_error_handler(error)
 
       private
 
+      # rubocop:disable Metrics/MethodLength
```

Comment:
Please don't disable good offense checks inline. Use rubocop_todo.yml.

Reply:
@ericboehs The method length is exceeded because of the multi-line formatting of the hashes. So again, I think locally disabling this rule here is justified.

Reply:
@ericboehs Removed rubocop disable